src/js/events/events.js

```diff
@@ -70,6 +70,7 @@ define([], function() {
         JWPLAYER_PLAYLIST_LOADED: 'playlist',
         JWPLAYER_AUDIO_TRACKS: 'audioTracks',
         JWPLAYER_AUDIO_TRACK_CHANGED: 'audioTrackChanged',
+        JWPLAYER_PLAYBACK_RATE_CHANGED: 'playbackRateChanged',
```

Comment:
I would like us to start replacing these with ES6 constants and state the intent to deprecate `jwplayer.events`.